chirp/drivers/ft450d.py

```diff
@@ -451,13 +494,14 @@ def _clone_in(self):
             else:
                 repeat = 1
             for _i in range(0, repeat):
-                data += self._read(block, blocks)
-                self.pipe.write(chr(CMD_ACK))
+                chunk = self._read(block, blocks)    # returns bytes()
+                data += chunk
+                self.pipe.write(bytes(chr(CMD_ACK), encoding='utf8'))
                 blocks += 1
                 status.cur = blocks
                 self.status_fn(status)
-        data += self.MODEL      # Append ID
-        return memmap.MemoryMap(data)
+        data += bytes(self.MODEL, encoding='utf8')      # Append ID
+        return memmap.MemoryMapBytes(data)

```

Comment:
This is good, but you need to set `NEEDS_COMPAT_SERIAL=False` on your driver if you do this, because the UI uses that flag to determine whether or not to give you a `MemoryMapBytes` or not when loading from a file. The two need to be consistent.